bones/dateBone.py

```diff
@@ -158,7 +158,7 @@ def fromClient( self,valuesCache, name, data ):
 		else:
 			err = self.isInvalid(value)
 			if not err:
```

Comment:
After further investigation i would recommend reverting these two changes and change this line to "if not err or value is None". This should do the trick and be more consistent with other bones

Reply:
OK make it so!